keyboards/dz60/keymaps/split_space_arrows/keymap.c

```diff
@@ -0,0 +1,78 @@
+#include "dz60.h"
+
+/* DZ60 layout using following options (from layouts diagram on KBDfans):
+ *  - plate B (2.25u lshift)
+ *  - opt 4 (1.75, 1, 1 on rshift)
+ *  - opt 10 (2.75, 1.25, 2.25 on space, 5x1u bottom right keys)
+ * http://www.keyboard-layout-editor.com/#/gists/225f0f4dcf6671405f744fabe314627c
+ */
+
+#define LAYOUT_b_4_10( \
+        K000, K001, K002, K003, K004, K005, K006, K007, K008, K009, K010, K011, K012,       K014, \
+        K100,       K102, K103, K104, K105, K106, K107, K108, K109, K110, K111, K112, K113, K114, \
+        K200,       K202, K203, K204, K205, K206, K207, K208, K209, K210, K211, K212, K213,       \
+        K300,       K302, K303, K304, K305, K306, K307, K308, K309, K310,       K312, K313, K314, \
+        K400, K401,       K403,       K404, K406,       K408,       K410, K411, K412, K413, K414  \
+        ) { \
+    { K000,  K001,  K002,  K003,  K004,  K005,  K006,  K007,  K008,  K009,  K010,  K011,  K012,  KC_NO, K014 }, \
+    { K100,  KC_NO, K102,  K103,  K104,  K105,  K106,  K107,  K108,  K109,  K110,  K111,  K112,  K113,  K114 }, \
+    { K200,  KC_NO, K202,  K203,  K204,  K205,  K206,  K207,  K208,  K209,  K210,  K211,  K212,  K213,  KC_NO }, \
+    { K300,  KC_NO, K302,  K303,  K304,  K305,  K306,  K307,  K308,  K309,  K310,  KC_NO, K312,  K313,  K314 }, \
+    { K400,  K401,  KC_NO, K403,  K404,  KC_NO, K406,  KC_NO, K408,  KC_NO, K410,  K411,  K412,  K413,  K414 }  \
+}
+
+// Mac vs Linux/windows behaviour for media next/prev keys
+#define IS_MAC 1
+#ifdef IS_MAC
+#define MEDIA_FWD KC_MFFD
+#define MEDIA_RWD KC_MRWD
+#elif
+#define MEDIA_FWD KC_MNXT
+#define MEDIA_RWD KC_MPRV
+#define
+#endif
```

Comment:
`KC_MNXT` and `KC_MPRV` both work on macOS; if you assign those keycodes you don't need this logic at all.